<ui version="4.0" >
 <class>ComprehensionPassageItem</class>
 <widget class="QWidget" name="ComprehensionPassageItem" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>299</width>
    <height>323</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Form</string>
  </property>
  <widget class="QLabel" name="m_lblPassageText" >
   <property name="geometry" >
    <rect>
     <x>10</x>
     <y>50</y>
     <width>265</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text" >
    <string>Passage</string>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget" >
   <property name="geometry" >
    <rect>
     <x>20</x>
     <y>290</y>
     <width>251</width>
     <height>25</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" >
    <item>
     <widget class="QPushButton" name="m_exitquiz" >
      <property name="text" >
       <string>Exit </string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="m_startquiz" >
      <property name="text" >
       <string>Questions</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="Line" name="line" >
   <property name="geometry" >
    <rect>
     <x>10</x>
     <y>40</y>
     <width>265</width>
     <height>16</height>
    </rect>
   </property>
   <property name="orientation" >
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QLabel" name="m_lblQuizName" >
   <property name="geometry" >
    <rect>
     <x>10</x>
     <y>20</y>
     <width>265</width>
     <height>14</height>
    </rect>
   </property>
   <property name="text" >
    <string>Quiz name </string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
